resources/deb/etc/default/openhab2

```diff
@@ -1 +1,12 @@
+# openHAB 2 service configuration
+# 
+# Configure additional options for the JAVA_OPTS environment variable.
+# These will be appended to the execution of the openHAB Java runtime in front of all other options.
+# 
+# EXTRA_JAVA_OPTS="-Dgnu.io.rxtx.SerialPorts=/dev/ttyUSB0:/dev/ttyS0:/dev/ttyS2:/dev/ttyACM0:/dev/ttyAMA0"
+# EXTRA_JAVA_OPTS="-Djna.library.path=/lib/arm-linux-gnueabihf/ -Duser.timezone=Europe/Berlin -Dgnu.io.rxtx.SerialPorts=/dev/ttyS0"
```

Comment:
An additional EXTRA_JAVA_OPTS after the statement in line 4 makes it seem like un-commenting both of these will append each. In reality it would only append the last line.

Could you add something along the lines of "A couple of independent examples:"